cloud_chatops/src/get_github_prs.py

```diff
@@ -22,29 +25,58 @@ def __init__(self, repos: List[str], owner: str):
         """
         self.repos = repos
         self.owner = owner
+        self._http_handler = HTTPHandler()
 
-    def run(self) -> Dict[str, List]:
+    def run(self) -> List[PrData]:
         """
         This method is the entry point to the class.
-        It runs the HTTP request methods and the formatting methods.
+        It runs the HTTP request methods and returns the responses.
         :return: The responses from the HTTP requests.
         """
-        unformatted_responses = self.request_all_repos_http()
-        formatted_responses = self.format_http_responses(unformatted_responses)
-        return formatted_responses
+        responses = self._request_all_repos_http()
+        return self._responses_to_dataclasses(responses)
 
-    def request_all_repos_http(self) -> Dict[str, List[Dict]]:
+    def _request_all_repos_http(self) -> List[Dict]:
         """
         This method starts a request for each repository and returns a list of those PRs.
         :return: A dictionary of repos and their PRs.
         """
-        responses = {}
+        responses = []
         for repo in self.repos:
             url = f"https://api.github.com/repos/{self.owner}/{repo}/pulls"
-            response = self.get_http_response(url)
-            responses[repo] = response
+            responses += self._http_handler.make_request(url)
         return responses
 
+    @staticmethod
+    def _responses_to_dataclasses(responses: List[Dict]) -> List[PrData]:
```

Comment:
copied over from #143 - mising docstring and function could be named better - something like `parse_responses` or `build_responses`